webapp/mapTest4.html

```diff
@@ -342,16 +342,87 @@
               .addTo(map); 
               }
               
-              function pressBtn9(){
-              var u = document.getElementById('button_press9').value;
-			  map.flyTo({center: [-122.284331,37.518310],speed: 0.3});
+              function pressBtnA(){
+              var u = document.getElementById('button_pressA').value;
+              map.flyTo({center: [-122.284934,37.517124],speed: 0.3});
 			  var popup = new mapboxgl.Popup({closeOnClick: true})
-              .setLngLat([-122.284331,37.518310])
-              .setHTML('<h3>Walter Gleason Gym</h3><p>The gym is equipped with a weight room, an athletic training room, and men’s and women’s locker rooms.</p>')
-              .addTo(map); 
+              .setLngLat([-122.284934,37.517124])
+              .setHTML('<h3>Campus Center</h3><p>The Campus center is comprised of the Academic Success Center, Admissions Office, International Student Office, and Student Life and Leadership Office</p>')
+              .addTo(map);
+              }
+              
+              function pressBtnB(){
+              var u = document.getElementById('button_pressB').value;
+              map.flyTo({center: [-122.284934,37.517124],speed: 0.3});
+			  var popup = new mapboxgl.Popup({closeOnClick: true})
+              .setLngLat([-122.284934,37.517124])
+              .setHTML('<h3>Academic Success Center</h3><p>The goal of the Academic Success Center is to support students in all aspects of their academic careers at NDNU, but it is up to the student to seek out these support services. The staff engages in a partnership with students to help them achieve their goals. Students are encouraged to investigate the programs and services to consider how the Center’s staff can assist them to address their concerns and realize their goals. The Academic Success Center offers you integrated learning and support services. Professional staff members, peer tutors and faculty work together to promote a supportive learning environment. Don’t hesitate to ask for help! We have all kinds of support services available.</p>')
+              .addTo(map);
+              }
+              
+              function pressBtnC(){
```

Comment:
I need to look at the rest of the JS code but are you guys not indenting functions?  While there is no technical reason to indent it makes everything much more readable is it is a part of every coding standard.  

ie.
`function foo() {
    var bar = 'zed';
}`